src/lightning_app/api/http_methods.py

```diff
@@ -59,6 +59,7 @@ async def fn(*args, **kwargs):
                     request_cls(
                         name=self.component_name,
                         method_name=self.method_name,
+                        timestamp=time.time(),
```

Comment:
What is the timestamp added for? Doesn't seem to be used

Reply:
Removed !